### ISSUE TITLE
Shared/Central Restyler configuration

### ISSUE THREAD

**Issue by iphydf** [restyled-io/restyler#96]
I can probably use a single restyled config for all 30+ repos in the TokTok org. It would be nice if I could share that single config in the .github repository.

Also, many other integrations have converged on putting their config in the .github directory. This is also nice to avoid having lots of dotfiles in the project root.

Note: nice to have, I don't *need* this feature. https://github.com/restyled-io/restyled.io/issues/197 is top for me :).

Unrelated: which repo should feature request issues be made on?

**Comment by pbrisbin** [comment]
I'm happy to implement checking a few places for a config (so look at `.github/restyled.y(a)ml` in addition to `.restyled.yaml`). I'd probably consider that it's own (minor) feature request distinct from the org-level part.

I'm not familiar how `org/` inheritance works generally. Is there a special "fake" repository where I'd curl `raw.github.com/$org/$something/.github/restyled.yaml` or something like that?

I'm also curious if you think piggy-backing on `remote_files` is an option for that...

```yaml
remote_files
  - url: https://.../restyled.yaml
    path: .restyled.yaml
```

A config of this and only this could run and replace itself before proceeding.

Or maybe we should just make it very obvious that it's something different, that's handled before anything else.

```yaml
replace: htps://.../restyled.yaml
```

For context, I'm trying to balance a few things:

- Good user ergonomics, but
- Don't be GitHub-lock-in (so don't hard-code a magic org-lookup)
- Simple implementation, by either:
  - A small extension to existing functionality (`remote_files`), or
  - Make this something very-very simple that we could just do in the `ENTRYPOINT` and not actually make any code changes to `restyler` itself.

> Unrelated: which repo should feature request issues be made on?

In short, it doesn't matter. Ideally, they would go in whichever repo those features might be implemented in, which is 99% of the time `restyler` or `restylers`. But I don't expect users to know or care about that level of technical detail, so reporting in `restyled.io` is fine and I can house-keep as necessary.

For an overview of what repos are responsible for what functionality, there is this page: https://github.com/restyled-io/restyled.io/wiki/Contributing-to-Restyled#repositories

**Comment by pbrisbin** [comment]
I extracted the simpler #98 so we can just keep this Issue for Shared/Central/Org-level configuration support.
